### DISCUSSION TITLE
I get an error when deploying to vercel. Help me.

### DISCUSSION THREAD

**Discussion by bukamasedo** [vercel/next.js#23818] [Help]
I get an error when deploying to vercel. Help me.

<img width="1305" alt="スクリーンショット 2021-04-08 21 54 34" src="https://user-images.githubusercontent.com/81907501/114030012-155f4600-98b5-11eb-9e98-9cd31a6c390e.png">

21:52:00 | Error: Command "npm run build" exited with 1
-- | --

**Comment by bukamasedo** [comment]
[https://github.com/Shu1512/shu-next-blog/tree/main/nextjs-blog](url)
Repository

**Comment by samrobbins85** [comment]
Your `Layout.js` has a capital, but you reference it with lowercase, and so it can't find the file

**Reply by bukamasedo** [reply]
ありがとうございます。thenk you
